app/keystore.properties

```diff
@@ -0,0 +1,4 @@
+storePassword=123456
```

Comment:
we shouldb't have keys in the repository --- it's public!

Reply:
key was removed in last commit, it didn't work anyway as I need the original signing key to package an APK for GooglePlay